### PR TITLE
GH-6452 Migrate installed_command.jsx to be pure and use Redux

### PR DESCRIPTION
**Pull request by 94117nl**
Summary:
Migrate installed_command.jsx to be pure and use Redux
Add snapshot test for InstalledCommand component

Issue Link:
https://github.com/mattermost/platform/issues/6452

### CODE REVIEW

File: webapp/tests/components/integrations/installed_command.test.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import * as Utils from 'utils/utils.jsx';
+import InstalledCommand from 'components/integrations/components/installed_command.jsx';
+
+describe('components/integrations/InstalledCommand', () => {
+    const emptyFunction = jest.fn();
+    const command = {
+        id: Utils.generateId(),
```
Comment (jwilander) [inline webapp/tests/components/integrations/installed_command.test.jsx]: Generating an id is causing the test to fail, it needs to be a constant (can be anything) so the snapshot gets the same value every time

Comment (94117nl) [inline webapp/tests/components/integrations/installed_command.test.jsx]: Got it, haven't had time to look into this yet

Comment (94117nl) [inline webapp/tests/components/integrations/installed_command.test.jsx]: Fixed failing tests

**Comment by hmhealey** [comment]
@94117nl Thanks for fixing that! Nice to have the "splash commands" fixed as well :)

**Pull request merged**